modules/smart-agent_kubernetes-common/detectors-kubernetes-common.tf

```diff
@@ -110,7 +110,7 @@ resource "signalfx_detector" "oom_killed" {
   authorized_writer_teams = var.authorized_writer_teams
 
   program_text = <<-EOF
-    signal = data('kubernetes.container_ready', filter=filter('container_status', 'terminated') and filter('container_status_reason', 'OOMKilled') and ${module.filter-tags.filter_custom})${var.oom_killed_aggregation_function}${var.oom_killed_transformation_function}.count().publish('signal')
+    signal = data('kubernetes.container_ready', filter=filter('container_status', 'terminated') and filter('container_status_reason', 'OOMKilled') and ${module.filter-tags.filter_custom})${var.oom_killed_aggregation_function}${var.oom_killed_transformation_function}.count().max(over='1m').publish('signal')
```

Comment:
I prefer you use the available `oom_killed_transformation_function` variable and set a default value instead of hardcode this here